docs/DevelopmentTips.md

```diff
@@ -33,9 +33,11 @@ Compilation times for the compiler and the standard library can be agonizing, es
 ```
 $ brew install sccache
 $ sccache --start-server
-$ ./swift/utils/build-script MY_ARGS --cmake-c-launcher $(which sccache) --cmake-cxx-launcher $(which sccache)
+$ ./swift/utils/build-script MY_ARGS --sccache
 ```
 
+If you want to always use sccache, you can `export USE_SCCACHE=1` and the build script will pick it up.
```

Comment:
should env vars be prefixed? `SWIFT_USE_SCCACHE`?

Reply:
I was copying the naming scheme used by the environment variable `distcc`, but I wouldn't be opposed to prefixing it.

Reply:
Yeah, I figured it was as good of timing as any to ask.